ui/pages/_manufacturerKey/_fixtureKey.vue

```diff
@@ -138,6 +139,10 @@
         <a href="/about#contact" class="card"><app-svg name="email" class="left" /><span>Contact</span></a>
       </div>
     </section>
+
+    <app-fixture-help-wanted-dialog
+      ref="helpWantedDialog"
```

Comment:
Do you need this ref? Otherwise, it should be deleted to make everything easier to understand.